src/main/java/ch/naviqore/raptor/QueryConfig.java

```diff
@@ -0,0 +1,57 @@
+package ch.naviqore.raptor;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * Configuration to query connections or iso-lines. Default values are set to infinity or zero, which means that no
+ * restrictions are set.
+ */
+@NoArgsConstructor
+@Getter
+public class QueryConfig {
+
+    private static final int INFINITY = Integer.MAX_VALUE;
+
+    private int maximumWalkingDuration = INFINITY;
```

Comment:
To discuss: I would make those final, or do we want to enable the configuration of the connection query config after creation? What would be the bebefits of this? 

An immutable config could be more consistent, which means i want to change something in the config, i need to create a new config.

Then instead of calling the setters in the constructor, call checks and set the values.

Reply:
Since the default values are set to not affect the "normal"/default unconstrained raptor. I prefer the implementation where you only need to set the settings you really want to set. Instead of having to specify that maximum travel duration and maximum number of transfers are Infinity only to set the max walking duration to 30 minutes.